### PR TITLE
Filesize Checks

### PR DESCRIPTION
**Pull request by JakeDawkins**
**SYS-3764**

# Feature / Fixed Issue(s)
- This checks the gzipped filesizes.
- checks files in `/web/dist/client`. I _think_ that's the right folder.
- right now, it checks all files in the folder the same way

### CODE REVIEW

**Comment by newspringweb** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  No Changelog changes!
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by richarddubay** [comment]
My local build files are about the same sizes as the ones that are listed in the danger comment above. So I would _assume_ that this is working correctly.

**Comment by jbaxleyiii** [comment]
@JakeDawkins this is a great start! I'd prefer to see the sizes listed in a table if possible. For instance each file that is over size could get a warning / failure, the name of the file (you can actually even generate a github link to it (I can show you how if you need to), the budget, and the size.

I'd love to get gzip into this PR before we merge it, but if need to do another task for it I'm good with that.

Let me know if you have any questions!

File: dangerfile.js
Comment (jbaxleyiii) [inline dangerfile.js]: @JakeDawkins for the links you can use https://github.com/NewSpring/Rock-Native/blob/1aefbed9bd96c76a0aa7b3a7fdb0ed5b65fc62b3/dangerfile.js#L8-L15

File: dangerfile.js
Comment (jbaxleyiii) [inline dangerfile.js]: we should gzip these prior to merging this PR

File: dangerfile.js
Comment (jbaxleyiii) [inline dangerfile.js]: 😍 I love this code

**Review by jbaxleyiii — CHANGES_REQUESTED** [review]

**Review by jbaxleyiii — APPROVED** [review]

**Comment by jbaxleyiii** [comment]
gzipping setup, but the `client.js` and `vendor.js` files it is finding shouldn't actually exist?

**Pull request merged**
